tests/unit/event_callback.c

```diff
@@ -0,0 +1,95 @@
+/* vim:expandtab:shiftwidth=2:tabstop=2:smarttab:
+ *
+ * Drizzle Client & Protocol Library
+ *
+ * Copyright (C) 2016 Andreas Bok Andersen (andreas.bok@sociomantic.com)
```

Comment:
Authorship and copyright are not the same, the author should be @andreas-bok-sociomantic but the copyright holder should be Sociomantic Labs GmbH, as long as this work has been done inside Sociomantic :)